collector/src/main/java/com/bitmovin/analytics/BitmovinAnalytics.java

```diff
@@ -206,7 +218,7 @@ public void onSeekComplete(long duration) {
         data.setDuration(duration);
         data.setVideoTimeStart(playerStateMachine.getVideoTimeStart());
         data.setVideoTimeEnd(playerStateMachine.getVideoTimeEnd());
-        sendEventData(data);
+        sendEventData(data, playerStateMachine.getCurrentState(), PlayerEvent.SEEKED);
```

Comment:
i'm in two minds about exposing the state and the event.. we are exposing a bunch of information to the features that might be useful but also might lead to unintended uses down the line if we are not careful.

Reply:
Ok, will remove that part completely for now.
But the only alternative I can think of for new would be something like hooking the features up to actual events like: `onBeforeExitErrorState`, `onBeforeExitBufferingState` etc. so we can set things on specific events. Which looks quite ugly, but you get the idea --> would look better once we restructure those `onExit` things.